CHANGELOG.md

```diff
@@ -7,26 +7,43 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Added
```

Comment:
Did you use some linter here? Perhaps we should add it to the CI so we don't crash on release?

Reply:
I think that should be a separate PR, it will include ignoring tests when *.md files are changed and a linter

Reply:
Lets add it to this PR, since if we don't, a typo in the changelog could break our release flow at a very late stage.

Reply:
okay sure